apps/experiments/models.py

```diff
@@ -566,13 +566,18 @@ class AgentTools(models.TextChoices):
     MOVE_SCHEDULED_MESSAGE_DATE = "move-scheduled-message-date", gettext("Move Reminder Date")
     UPDATE_PARTICIPANT_DATA = "update-user-data", gettext("Update Participant Data")
     ATTACH_MEDIA = "attach-media", gettext("Attach Media")
+    END_SESSION = "end-session", gettext("End Session")
 
     @classmethod
     def reminder_tools(cls) -> list[Self]:
         return [cls.RECURRING_REMINDER, cls.ONE_OFF_REMINDER, cls.DELETE_REMINDER, cls.MOVE_SCHEDULED_MESSAGE_DATE]
 
+    @classmethod
+    def legacy_experiment_tools(cls):
```

Comment:
Since the legacy tool group is only used in the tests, how do you feel about making this a fixture or something? I was initially confused by why we make a distinction between these tools and the "pipeline specific" tools, but looking at the test it makes sense.